sdk/clientcore/core/src/main/java/io/clientcore/core/implementation/http/rest/SwaggerMethodParser.java

```diff
@@ -148,6 +149,38 @@ public SwaggerMethodParser(Method swaggerMethod) {
             }
         }
 
+        final String[] requestQueryParams = httpRequestInformation.queryParams();
+
+        if (requestQueryParams != null) {
+            for (final String queryParam : requestQueryParams) {
+                final int equalsIndex = queryParam.indexOf("=");
+
+                if (equalsIndex >= 0) {
+                    final String paramName = queryParam.substring(0, equalsIndex).trim();
+
+                    if (!paramName.isEmpty()) {
+                        String paramValue = queryParam.substring(equalsIndex + 1).trim();
+
+                        if (!paramValue.isEmpty()) {
+                            constantQueryParams.compute(paramName, (key, value) -> {
+                                if (value == null) {
+                                    List<String> valueList = new ArrayList<>();
+
+                                    valueList.add(paramValue.replace(",", "%2C"));
```

Comment:
We'll want to use proper query parameter value encoding here, should replace this with `UrlEscapers.QUERY_ESCAPER.escape`